remappings.txt

```diff
@@ -1,3 +1,3 @@
 ds-test/=lib/forge-std/lib/ds-test/src/
 forge-std/=lib/forge-std/src/
-openzeppelin-contracts/=lib/openzeppelin-contracts/
+redstone-oracles-monorepo/=lib/redstone-contracts/src/RedStoneBaseContracts/redstone-oracles-monorepo
```

Comment:
do you think naming it like `redstone-oracles-monorepo` makes sense? prehaps `redstone-oracle`?